lib/send.js

```diff
@@ -18,5 +18,6 @@ module.exports = function send (event, callback) {
     name: event.name || "", // allow email to be sent without name
     pixel_url: "https://" + process.env.EMAIL_APP_URL + "/read/" + token
   };
-  return sendemail(template, options, callback);
+  const merged = {...event, ...options}; // expose all of event to template
```

Comment:
see how to use the [spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax)